### DISCUSSION TITLE
Get Scala version Exception with setting FLINK_HOME

### DISCUSSION THREAD

**Discussion by wjcwin** [apache/streampark#2385] [Q&A]
streampark version:2.0.0 , mixed mode ,scala-2.11
local flink version: 1.13.1_2.11

Detail exception:
java.lang.UnsupportedOperationException: The current Scala version of StreamPark is 2.11.12, but the scala version of Flink to be added is 2.11-1.13, which does not match, Please check

**Comment by Al-assad** [comment]
You can switch to the release version of Flink Scala 2.12 release. StreamPark 2.0 default release version is compiled with Scala 2.12.
